### PR TITLE
Implement Accomplishments & Ability to Give Accomplishments to Members

### PR DESCRIPTION
**Pull request by pulchroxloom**

### CODE REVIEW

File: app/controllers/points_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    points_valid_member
+  end
+
+  def points_valid_member
+    @member = Member.find_by_uid(session[:app_user]['uid'])
+    unless @member.paid_dues?
+      flash[:alert] = 'You must pay dues to see your points.'
+      redirect_to('/dashboard')
+    end
+  end
```
Comment (shivaniiyer) [inline app/controllers/points_controller.rb:27]: I see you added an index and points valid! Nice thinking on making sue their dues have been paid

File: app/views/accomplishments/show.html.erb [hunk 2]
```diff
@@ -6,27 +6,31 @@
...
 		<th>Added to Database on</th>
-		<td><%= @accomplishment.created_at %></td>
+		<td><%= @accomplishments.created_at %></td>
 	</tr>
 	<tr>
 		<th>Last Updated on</th>
-		<td><%= @accomplishment.updated_at %></td>
+		<td><%= @accomplishments.updated_at %></td>
 	</tr>
 </table>
```
Comment (shivaniiyer) [inline app/views/accomplishments/show.html.erb:36]: I see you added a is_dues field. nice touch

File: app/views/points/accomplishments.html.erb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+				<td><%= member.accomplishments.points %></td>
+				<td>
+					<%= link_to member.accomplishments_path(member.accomplishments) do %>
+						<i class="fa fa-info-circle"></i>
+					<% end %>
+				</td>
+			</tr>
+		<% end %>
+	</tbody>
+</table>
```
Comment (shivaniiyer) [inline app/views/points/accomplishments.html.erb:30]: so details just links It to member accomplishments? what all does that show?

**Review by shivaniiyer — APPROVED** [review]
I see you added a new field in accomplishments and fixed up the points one too. Looks good to me

File: app/controllers/concerns/new_email.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    @member = Member.find_by_email(@supplied_email)
+
+    if (@member)
+      session[:app_user]['kicked_out'] = true
+    end
+
+    redirect_to '/logout', alert: 'Signup Failed: There is already an account with this email address' if @member
+    #redirect_to dashboard_path, notice: 'You have already signed up!' if @member
+  end
+end
```
Comment (dvansteen) [inline app/controllers/concerns/new_email.rb:25]: Probably not a big deal but github and rubocop will complain

Comment (pulchroxloom) [reply app/controllers/concerns/new_email.rb:25]: I didn't consciously edit this. I think if you `git blame` I shouldn't show up. We will let RuboCop fix it later.

**Review by timaeus321 — CHANGES_REQUESTED** [review]
Blocking Dantes code

**Comment by pulchroxloom** [comment]
Merge branch should not be `master`. This is a mistake.

File: app/views/accomplishments_members/delete.html.erb [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+	<tr>
+		<th>Last Updated on</th>
+		<td><%= @accomplishments_member.updated_at %></td>
+	</tr>
+</table>
+
+<%= form_for(@accomplishments_member, :method => 'delete') do |f| %>
+	<p>Are you sure?</p>
+	<%= f.submit("Delete #{@accomplishments_member.member.name}'s #{@accomplishments_member.accomplishment.name}", :class => "btn btn-primary") %>
+<% end %>
```
Comment (dvansteen) [inline app/views/accomplishments_members/delete.html.erb:59]: This looks like tabs

**Pull request closed without merging**